code/game/gamemodes/changeling/powers/mutations.dm

```diff
@@ -247,7 +247,7 @@
 
 
 /obj/item/gun/magic/tentacle/shoot_with_empty_chamber(mob/living/user as mob|obj)
-	to_chat(user, "<span class='warning'>The [name] is not ready yet.<span>")
+	to_chat(user, "<span class='warning'>The [name] is not ready yet.</span>")
```

Comment:
name -> src

Reply:
The the [name], eh?

Reply:
What would be correct here? "\The [name]"?

Reply:
either `The [name]` or `[src]`